.travis.yml

```diff
@@ -0,0 +1,58 @@
+language: python
+python:
+  - "3.6"
+
+install:
+  - sudo apt-get update
+#   - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
+#       wget https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh -O miniconda.sh;
+#     else
+#       wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+#     fi
+#   - bash miniconda.sh -b -p $HOME/miniconda
+#   - export PATH="$HOME/miniconda/bin:$PATH"
+#   - hash -r
+#   - conda config --set always_yes yes --set changeps1 no --set auto_activate_base False
+#   - conda update -q conda
+#   - conda init
+#   - source activate 
+#   - conda info -a
+#   
+# #   - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION
+# #   - source activate test-environment
+#   - conda config --add channels conda-forge
+#   - conda config --add channels bioconda
+#   - conda config --add channels anaconda
+#   - conda config --add channels workflowconversion
+# 
+#   - conda install python=$TRAVIS_PYTHON_VERSION
+#   # TODO could be replaced by conda package once it is merged and released
+#   - git clone -b topic/no-1-2x https://github.com/bernt-matthias/CTDopts $HOME/CTDopts
+#   - cd $HOME/CTDopts
+#   - python setup.py install 
+#   - conda install -c conda-forge lxml
+#   - conda install -c conda-forge ruamel.yaml 
+#   - conda install libxml2
+#   - cd $TRAVIS_BUILD_DIR
+#   - python setup.py install
+#   - conda install coverage green codecov
+
+#   - virtualenv .venv
+#   - . .venv/bin/activate
+  - pip install git+https://github.com/bernt-matthias/CTDopts@topic/no-1-2x
```

Comment:
`WorkflowConversion:master is up to date with all commits from bernt-matthias:topic/no-1-2x.`

But the reverse is not true, and is the source of at least one of the failing tests

https://github.com/bernt-matthias/CTDopts/compare/topic/no-1-2x...WorkflowConversion:master#diff-a55c430e446e6696bb3f110ad5ce8607e83273a276ce6aaef8816a285b014503R50